apps/rush-lib/src/logic/Git.ts

```diff
@@ -1,57 +1,74 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+import child_process from 'child_process';
```

Comment:
Might make sense to reexport `SpawnSyncReturns<T>` from `@rushstack/node-core-library`?

Reply:
Yeah probably a good idea. I'd rather do that in a separate PR.